projects/ccf-database/src/lib/hubmap/hubmap-data.ts

```diff
@@ -326,7 +326,10 @@ export class HuBMAPTissueBlock {
   }
 
   getDatasetThumbnail(dataset: JsonDict, assetsApi: string, serviceToken?: string): string | undefined {
-    if (dataset.group_uuid === '73bb26e4-ed43-11e8-8f19-0a7c1eab007a') { // TMC-Vanderbilt
+    if (dataset.thumbnail_file) {
+      const uuid = (dataset.thumbnail_file as JsonDict).file_uuid;
+      return `${uuid}/thumbnail.jpg` + (serviceToken ? `?token=${serviceToken}` : '');
+    } else if (dataset.group_uuid === '73bb26e4-ed43-11e8-8f19-0a7c1eab007a') { // TMC-Vanderbilt
```

Comment:
Did any datasets fall through to the other cases?

Reply:
Yes some of them don't have a thumbnail file.

Reply:
That's not the question

Reply:
What do you mean by other cases

Reply:
Were there any cases within that function where there was no thumbnail_file, but a thumbnail was found in the other two else clauses?

Reply:
No it doesn't look like it.